src/iris/ui/auth/ldap.py

```diff
@@ -10,16 +10,21 @@ class Authenticator:
     def __init__(self, config):
         root = os.path.abspath('./')
         self.ldap_url = config['auth']['ldap_url']
-        self.cert_path = os.path.join(root, config['auth']['ldap_cert_path'])
+        self.start_tls = config['auth']['start_tls']
```

Comment:
Switch to config['auth'].get('start_tls') to maintain existing behavior without need for config change

Reply:
^ agreed :)